app/server/src/generator/templates/template6/index.js

```diff
@@ -34,12 +34,9 @@ function generateProfileSection(profile) {
     if (names.length === 1) {
       nameStart = names[0]
       nameEnd = ''
-    } else if (names.length === 2) {
-      nameStart = names[0]
-      nameEnd = names[1]
     } else {
-      nameStart = names.slice(0, names.length - 1).join(' ')
-      nameEnd = names[names.length - 1]
+     nameStart = names[0]
+     nameEnd = names.slice(1,names.length).join(' ')
     }
   }
 
```

Comment:
Same as before, just make sure the indentation/whitespace is consistent with the project.